microdata_validator/dataset_reader.py

```diff
@@ -91,7 +92,7 @@ def _read_and_process_data(data_file_path: Path,
                     value = data_row[1].strip('"')
                     start = data_row[2].strip('"')
                     stop = data_row[3].strip('"')
-                    # TODO: attributes = data_row[4]
+                    # attributes = data_row[4]
```

Comment:
Why is it commented out? I would like a description here

Reply:
It is not really on the plan right now to validate additional attributes, so it felt weird to have a TODO for something we might not do. I could remove these comments to like you mention below? That seems fair 👍🏻

Reply:
Right, removing them will make the intent clearer I think 👍